test/test_fx_passes.py

```diff
@@ -310,12 +333,52 @@ def test_fuser_util(self, partition):
         fused_graph = fuse_by_partitions(gm, partitions)
 
         a, b, c = torch.rand(4), torch.rand(4), torch.rand(4)
-
         expected = m(a, b, c)
         result = fused_graph(a, b, c)
 
         torch.testing.assert_close(expected, result)
 
+    @parametrize("partition", [
+        [['conv1', 'conv2', 'add_2', 'conv3']],
+    ])
+    def test_fuser_util_with_get_attr(self, partition):
```

Comment:
nitpick, this is kinda hard to read what it's trying to test... 
Do you really need to construct a quantized model for testing the get_attr copying behavior?